### PR TITLE
Add relations

### PR DESCRIPTION
**Pull request by IvanAndrukh**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=738)

## Code reviewers

- [x] @dmytrorevak
- [x] @IvanAndrukh
- [x] @JjuliaP
- [x] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## Summary of issue

Add relations: CustomProfile - CustomUser

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/customprofile/models.py [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
         :param create_date: The date when the
                             certain profile was created.
         :type: create_date: DateTimeField
+
         :param update_date: The date when the certain
                             profile was last time edited.
         :type update_date: DateTimeField
-
     """
+    user = models.OneToOneField(CustomUser, on_delete=models.CASCADE, null=True)
```
Comment (makarman) [inline eventually/customprofile/models.py]: Can it be null=True?

File: eventually/customprofile/models.py [hunk 2]
```diff
@@ -46,16 +47,24 @@ def __str__(self):
         :return: profile hobby, profile photo, profile birthday
         """
 
-        return "{} {} {}".format(self.name, self.description, self.status)
+        return "{} {} {}".format(self.hobby, self.photo, self.birthday)
```
Comment (makarman) [inline eventually/customprofile/models.py]: add __repr__ method

File: eventually/customprofile/models.py [hunk 3]
```diff
@@ -125,17 +136,23 @@ def to_dict(self):
             'photo': self.photo,
             'birthday': self.int(self.birthday.timestamp()),
```
Comment (makarman) [inline eventually/customprofile/models.py]: Why self.int? Maybe just int

File: eventually/customprofile/models.py [hunk 4]
```diff
@@ -125,17 +136,23 @@ def to_dict(self):
...
             'created_at': self.int(self.created_at.timestamp()),
-            'updated_at': self.int(self.updated_at.timestamp())
+            'updated_at': self.int(self.updated_at.timestamp()),
+            'user': self.user.id
```
Comment (makarman) [inline eventually/customprofile/models.py]: if user can be None use self.user.id if self.user else None,

**Review by redmyt — APPROVED** [review]

**Pull request merged**